front/src/modules/ui/layout/board/components/BoardOptionsDropdownContent.tsx

```diff
@@ -194,6 +199,21 @@ export const BoardOptionsDropdownContent = ({
               text="Stages"
             />
           </DropdownMenuItemsContainer>
+          <DropdownMenuSeparator />
+          <DropdownMenuItemsContainer>
+            <MenuItemNavigate
```

Comment:
We should use a MenuItemToggle instead, and not change MenuItemNavigate behavior